apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -101,6 +103,16 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
                 return addValidationError(`${UNEQUAL_CONTEXT_ERROR}:property:context] Context is not equal to property.context`)
             }
         },
+        afterChange: async ({ context, updatedItem, operation }) => {
+            if (operation === 'create'){
```

Comment:
Why there's no update? According to accesses we can freely change billing accounts.

Reply:
Should old service consumers be deleted if unitName changes for example, or should hook be called and so on?

Reply:
Also what should be done if BillingPorperty address changed?

Reply:
There will be no cases when address for billingProperty will change
And discoverService consumer will not watch for serviceConsumers after it creates them

Reply:
we want discoverServiceConsumer to work only on new residents and new billing accounts
it's not an universal tool for managing service consumers for resident 
it's only a helper to get receipts for the first time

Reply:
Should we block update access on billing entities then?